src/hooks/Tokens.ts

```diff
@@ -248,12 +248,14 @@ export function searchInactiveTokenLists({
     if (!list) continue
     for (const tokenInfo of list.tokens) {
       if (tokenInfo.chainId === chainId && tokenFilter(tokenInfo)) {
-        const wrapped: WrappedTokenInfo = new WrappedTokenInfo(tokenInfo, list)
-        if (!activeTokens[wrapped.address] && !addressSet[wrapped.address]) {
-          addressSet[wrapped.address] = true
-          result.push(wrapped)
-          if (result.length >= minResults) return result
-        }
+        try {
```

Comment:
why do you need try catch here?
just check: `if (isAddress(tokenInfo.address) && tokenInfo.chainId === chainId && tokenFilter(tokenInfo))`

Reply:
But :pepe-sad: đâu ai nói trước chữ ngờ

Reply:
requested change